### ISSUE TITLE
Unique ID for contact form submissions

### ISSUE THREAD

**Issue by JakeVolpto** [nciske/salesforce-wordpress-to-lead#100]
Hi! Is it possible to change the email (to the admin) subject name from a generic message to a custom message that includes a unique ID for every new submission made through the forms?
Thanks.

**Comment by nciske** [comment]
Yes. There’s a filter for the subject line documented in the reader.txt file. Just replace it with what you’d prefer.

**Comment by nciske** [comment]
What I meant to say was... I swear that filter already existed, but it does not appear to. So I’ll get that added in the next release!

**Comment by JakeVolpto** [comment]
Awesome that would be incredibly helpful! Thanks, any idea when that would be?

**Comment by nciske** [comment]
I was right, the docs are just out of sync with the code!

```
$subject = apply_filters('salesforce_w2l_cc_admin_email_subject', $subject, $form_type, $post );`
```

So something like this:

```
add_filter('salesforce_w2l_cc_admin_email_subject', 'salesforce_w2l_cc_admin_email_subject_example', 10, 3);
function salesforce_w2l_cc_admin_email_subject_example($subject, $form_type, $post ){
    return 'Your custom message ' . uniqid('SF');
}
```

**Comment by JakeVolpto** [comment]
Thank you so much, worked a charm!

**Comment by JakeVolpto** [comment]
Hi again, is there a way to target individual forms in the php as I want a custom subject message per form? Thanks!

**Comment by nciske** [comment]
Not yet - that filter is a bit limited. I've got per form filters and and a global filter with form_id passed to it working on my development branch. I need to clean up a few things then I can push a release.
